app/javascript/packs/billboard.js

```diff
@@ -0,0 +1,25 @@
+import { setupDisplayAdDropdown } from '../utilities/displayAdDropdown';
+
+// the term billboard can be synonymously interchanged with displayAd
+async function getBillboard() {
+  const placeholderElement = document.getElementsByClassName(
+    'js-display-ad-comments-container',
+  )[0];
+
+  const { articleId, placementArea } = placeholderElement.dataset || {};
+
+  if (placeholderElement.innerHTML.trim() === '') {
+    const response = await window.fetch(
+      `/display_ads/for_display?article_id=${articleId}&placement_area=${placementArea}`,
```

Comment:
I like that this is focused on just one placement-area, initially, likely you've already thought of this, but just in case: we could potentially provide the ad URL in a `data` attribute on the `container` (letting us manage all route composition server-side), and, conceivably, could use a `querySelectorAll` to find any/all `placeholderElements` with a `data-async-url` (or whatever we name it).

Reply:
@jaw6 thanks, yeah I think I'll take this next step and refactor this out when I do the "post-sidebar" display ad sync.

Reply:
Implemented this in this PR.